app/crud/crud_user.py

```diff
@@ -9,8 +9,11 @@
 
 
 class CRUDUser(CRUDBase[User, UserCreate, UserUpdate]):
-    def get_by_username(self, db: Session, *, username: str) -> Optional[UserCreate]:
+    def get_by_username(self, db: Session, *, username: str) -> Optional[User]:
         return db.query(User).filter(User.username == username).first()
+    
+    def get_by_email(self, db: Session, *, email: str) -> Optional[User]:
+        return db.query(User).filter(User.email == email).first()
```

Comment:
```suggestion
        return db.query(User).filter_by(email=email).one_or_none()
```
I think this is better. Same above